src/model/model_helper.py

```diff
@@ -27,6 +28,18 @@ def is_empty(value):
     return (not value) and (value != 0) and (value != False)
 
 
+def update_output_files_with_vars(output_files, arguments):
+    for i, output_file in enumerate(output_files):
+        regex = "\$\$([a-zA-Z0-9-_]+)"  # Syntaxe is : $$my_variable (with two '$')
+        matched_arguments = re.findall(regex, output_file)
+        for argument in matched_arguments:
+            if argument in arguments:
+                value = arguments[argument]
+                output_file = output_file.replace("$$" + argument, value)
+        output_files[i] = output_file
```

Comment:
Could we return modified array? Instead of changing internal state of the config?
Also I would suggest to move the function to _features/file_download_feature.py_ and call it directly inside `prepare_downloadable_files`

Reply:
Yes good idead, I will modified it.